### DISCUSSION TITLE
No implementation for minRegionArea use in a temp obstacles NavMesh?

### DISCUSSION THREAD

**Discussion by SpangMcgraw** [recastnavigation/recastnavigation#674] [Q&A] [answered]
So I was looking at a project that is using the TempObstacles NavMesh and we had a bunch of small islands, I was tweaking the minRegionArea value and noticed it was having no effect, I had a trace through the code and looked in the demo as well and noticed MinRegionArea is not used currently when building a TempObstacle NavMesh. Is this intentional or was it just forgotten?

**Comment by memononen** [comment, marked as answer]
I omitted it intentionally back then when I was working on the tile cache.

The small region pruning does not work across tile boundaries, since it does not know if the small region on this tile will actually be part of larger region on the other side of the tile.

The tiles made with temp obstacle tend to be a lot smaller, and small region pruning was not effective most of the time.
